.vscode/launch.json

```diff
@@ -0,0 +1,19 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Python: Current File",
+            "type": "python",
+            "request": "launch",
+            "program": "${file}",
+            "console": "integratedTerminal",
+            "justMyCode": true,
+            "envFile": "${workspaceFolder}/.env",
+            "preLaunchTask": "Source Shell"
+           
```

Comment:
This helps with debugging, I should add some local development instructions to the README.  
I wonder if I should switch to a .env file instead of env.sh, probably easier that way.  Or that's something for a future release.